src/main/kotlin/org/malachite/estella/services/InterviewService.kt

```diff
@@ -58,39 +58,36 @@ class InterviewService(
     fun getLastInterviewFromApplicationId(applicationId: Int): InterviewId =
             withExceptionThrower { interviewRepository.getAllByApplicationId(applicationId).sortedWith { a, b ->
                 a.dateTime?.compareTo(b.dateTime) ?: -1
-            }.first() }.toId()
+            }.first() }.getId()
 
     fun setHosts(id: UUID, hostsMails: List<String>) {
         if (!canHrUpdate(id)) throw UnauthenticatedException()
-        val interview = interviewRepository.findById(id)
-        interview.ifPresent {
-            val int = interviewRepository.save(it.copy(hosts=hostsMails))
-            val application = int.application
-            val offer = recruitmentProcessService.getProcessFromStage(application.stage).offer
-            hostsMails.forEach { mail -> mailService.sendInterviewDevInvitationMail(offer, int, application, mail) }
-        }
+        val interview = getInterview(id)
+        val savedInterview = interviewRepository.save(interview.copy(hosts=hostsMails))
+        val application = savedInterview.application
+        val offer = recruitmentProcessService.getProcessFromStage(application.stage).offer
```

Comment:
to wszystko można byłoby zrobić na let'ach (bardziej funkcyjnie, a wy się tym jaracie)
w stylu:
```kotlin
getInterview(id).let{
    interviewRepository.save(it.copy(...))
}.let {
    it.application
}.let {
   recruitmentProcessService....(it.stage).offer
}
```

Reply:
nie do końca, bo pod koniec jest 
```java 
hostsMails.forEach { mail -> mailService.sendInterviewDevInvitationMail(offer, savedInterview, application.application, mail) }
```
więc potrzebujemy tych 3 rzeczy w jednym miejscu